docs/pages/application-access/jwt/grafana.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Application Access with Grafana
+description: How to configure Teleport Application Access for Grafana.
+---
+
+This guide will help you configure Grafana [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/)
+with Teleport Application Access.
+
+## Prerequisites
+
+- Teleport cluster version >= `5.0.0` with running Auth/Proxy Services or
+  Teleport Cloud account.
+- Running [Application Service](../guides/connecting-apps.mdx).
+- Grafana cluster version >= `8.4.0`.
+
+## Step 1/3. Enable a JWT authentication in Grafana
+
+Update your Grafana configuration file to enable JWT authentication. This file
+is typically located at `/usr/share/grafana/conf/defaults.ini`.
```

Comment:
Is this the file we should be editing versus `/etc/grafana/grafana.ini`?

>The default settings for a Grafana instance are stored in the $WORKING_DIR/conf/defaults.ini file. Do not change this file.
>
>Depending on your OS, your custom configuration file is either the $WORKING_DIR/conf/defaults.ini file or the /usr/local/etc/grafana/grafana.ini file. The custom configuration file path can be overridden using the --config parameter.

From https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/